scripts/ontology_to_ontology_mapping_tool/ontology_to_ontology_mapping_tool.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
+#
+# This library is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF
+# MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.  The software and
+# documentation provided hereunder is on an "as is" basis, and
+# Memorial Sloan-Kettering Cancer Center
+# has no obligations to provide maintenance, support,
+# updates, enhancements or modifications.  In no event shall
+# Memorial Sloan-Kettering Cancer Center
+# be liable to any party for direct, indirect, special,
+# incidental or consequential damages, including lost profits, arising
+# out of the use of this software and its documentation, even if
+# Memorial Sloan-Kettering Cancer Center
+# has been advised of the possibility of such damage.
+
+import sys
+import os
+import io
+import argparse
+import pandas as pd
+import requests
+
+ONTOLOGY_MAPPINGS_FILE_URL = "https://raw.githubusercontent.com/cBioPortal/oncotree/master/scripts/ontology_to_ontology_mapping_tool/ontology_mappings.txt"
+VALID_ONCOTREE_CODES_URL = "https://raw.githubusercontent.com/cBioPortal/oncotree/master/resources/resource_uri_to_oncocode_mapping.txt"
+ACCEPTED_ONTOLOGIES = ['ONCOTREE_CODE','UMLS_CODE','NCIT_CODE','ICDO_CODE','HEMEONC_CODE']
+NULL_VALUES = ['','NA']
+
+def add_comments_column_and_log_data(mapped_data,target_file,source_code,target_code,source_file):
```

Comment:
throughout this file .. add a space character after comma in argument lists / function calls

Reply:
Fixed